pmd-scala-modules/pmd-scala-common/pom.xml

```diff
@@ -74,6 +74,16 @@
                     </executions>
                 </plugin>
 
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <configuration>
+                        <systemPropertyVariables>
+                            <mvn.project.src.test.resources>${project.basedir}/../pmd-scala-common/src/test/resources</mvn.project.src.test.resources>
```

Comment:
That should not be necessary, thanks to https://github.com/pmd/pmd/blob/e435037fdc5573414d5a34a02a9f7007869b7809/pom.xml#L259

I'll test, whether it works without this config...